miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/service/RunScannerClient.java

```diff
@@ -57,34 +65,14 @@
 
   private static final Pattern WHITESPACE = Pattern.compile("\\s+");
 
-  // This self-autowires so we can call the proxified version of ourselves to create transactions.
-  @Autowired
-  private RunScannerClient client;
   private final Semaphore lock = new Semaphore(1);
   @Autowired
   private RunService runService;
+  @Autowired
+  private SecurityManager securityManager;
   private final ConcurrentMap<String, ProgressiveRequestDto> servers = new ConcurrentHashMap<>();
 
-  public void process(List<NotificationDto> results) {
-    try (AutoCloseable timer = acquireTime.start()) {
-      lock.acquire();
-    } catch (Exception e) {
-      log.error("Failed to acquire lock", e);
-      return;
-    }
-    for (NotificationDto dto : results) {
-      try (AutoCloseable timer = saveTime.start()) {
-        (runService.processNotification(dto) ? saveNew : saveUpdate).inc();
-        saveCount.inc();
-      } catch (Exception e) {
-        log.error("Failed to save run: " + dto.getRunAlias(), e);
-        saveFailures.inc();
-      }
-    }
-    lock.release();
-  }
-
-  @Scheduled(fixedDelay = 5 * 60 * 1000)
+  @Scheduled(fixedDelayString = "${miso.runscanner.interval}")
```

Comment:
Use `"${miso.runscanner.interval:300000}"` to make the property optional (startup won't fail if the property is missing)